end-to-end-untargeted-metabolomics.Rmd

```diff
@@ -18,11 +18,25 @@ bibliography: references.bib
 ---
 
 ```{r style, warning=FALSE, include=FALSE, results='hide'}
-library("BiocStyle")
-library("knitr")
-library("rmarkdown")
-opts_chunk$set(message = FALSE, error = FALSE, warning = FALSE,
-               cache = FALSE)
+if (!require("BiocManager", quietly = TRUE))
```

Comment:
thanks for the suggestion @Danmarques29 , but I think it would be better to have this code in the README (or introduction) at a section such as "Requirements and installation instruction". If it's directly in the Rmd file in a code block it would be executed every time you run the workflow - which would re-install the packages every time.